caper/cromwell_metadata.py

```diff
@@ -109,7 +109,7 @@ def calls(self):
 
     @property
     def recursed_calls(self):
-        return self.recurse_calls(lambda _, call, __: call)
+        return list(self.recurse_calls(lambda _, call, __: call))
```

Comment:
Is this cacheable?

Reply:
Thanks, that's a good catch. reverted it and added docstring.